node/runtime/pangoro/src/lib.rs

```diff
@@ -829,7 +833,21 @@ impl dvm_rpc_runtime_api::ConvertTransaction<OpaqueExtrinsic> for TransactionCon
 }
 
 fn migrate() -> Weight {
-	<darwinia_staking::MinimumValidatorCount<Runtime>>::put(2);
+	if let Ok(bytes) = array_bytes::hex2bytes(
```

Comment:
@xiaoch05 Please help check about this migration content.

Reply:
LGTM

Reply:
cc @furoxr

Reply:
Encode from

```
{
	"difficulty": "0x2",
	"extraData": "0xd683010108846765746886676f312e3137856c696e757800000000005865ba3cd1d5f5f372bbfb5deeef7ce4b5637139b1a5d9a03e21496ddabac6a6e5d829a218ee28045873c8427cb5a77c2ce86815a3c110986c0a7ac15f560661e709ff9b01",
	"gasLimit": "0x1c7f9be",
	"gasUsed": "0x20f04d",
	"logsBloom": "0x0000100000000000044000000000000080000000001000000000400000000000100000000000000020000000100000000801040002400000000000000020000000000000000000100000080e000000002010000000000000000000800000400808080020020208000400000000000802084001000000080000000010000001000000000000200080000000000801000000000440000000000000000000900020020000020000000001000000000000000000042000000000000000000020000000000006000000008000000000000000002008000000000000000042000020400010000000000000610000000000008002000000000000080000000008000001",
	"miner": "0x1284214b9b9c85549ab3d2b972df0deef66ac2c9",
	"mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
	"nonce": "0x0000000000000000",
	"number": "0xfd2c2b",
	"parentHash": "0x14e7e1cf1be0318b419a611f02c024533a9e52083fab41ca7009e69312a3e4a3",
	"receiptsRoot": "0xd12bb98573b478b7210e7387a09a8323c288b7b989b06d808b997ba8b4124e61",
	"sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
	"stateRoot": "0x6e008b9b78dbb45a9c2725c94a2ed46d6a77e187d052fcc1225ac4b3087e8295",
	"timestamp": "0x62032bb0",
	"transactionsRoot": "0x4a884d96f75c303ba9bd9add83e8e2b55b91bad0a0a1f953ae529fb8226daa62"
}
```